tests/functional/adapter/test_iceberg_format.py

```diff
@@ -14,25 +14,21 @@
 
 import pytest
 
-from dbt.tests.util import (
-    run_dbt,
-    check_result_nodes_by_name,
-    relation_from_name,
-    check_relation_types,
-    check_relations_equal,
-)
-
-from tests.functional.adapter.test_basic import (
-    TestSimpleMaterializationsImpala,
-    TestIncrementalImpala,
-)
-
 from dbt.tests.adapter.basic.files import (
     model_base,
     model_incremental,
     base_view_sql,
     schema_base_yml,
 )
+from dbt.tests.adapter.basic.test_base import BaseSimpleMaterializations
+from dbt.tests.adapter.basic.test_incremental import BaseIncremental
+
+from dbt.tests.util import (
+    run_dbt,
+    check_result_nodes_by_name,
+    relation_from_name,
+    check_relation_types,
+)
 
 
 def is_iceberg_table(project, tableName):
```

Comment:
Can we add a function to make sure the created table is of format v2.

Reply:
The verification of table format-version is blocked by [https://issues.apache.org/jira/browse/IMPALA-12209,](https://issues.apache.org/jira/browse/IMPALA-12209) as impala currently doesn't provide the format-version in the output.